build/bazel/remote/execution/v2/remote_execution.proto

```diff
@@ -520,6 +520,49 @@ message Platform {
   // The server MAY use the `value` of one or more properties to determine how
   // it sets up the execution environment, such as by making specific system
   // files available to the worker.
+  //
+  // The following standard `name`s are defined:
+  //
+  // * `OS`: This describes the operating system. Only a single value is allowed
+  //   and an exact match is required. The following standard values are
+  //   defined:
+  //
+  //     - AIX
+  //     - FreeBSD
+  //     - Linux
+  //     - MacOS
+  //     - SunOS
+  //     - Windows
+  //
+  // * `ISA`: This describes the instruction set architecture including
+  //   instruction set extensions. Multiple values are allowed. The worker is
+  //   required to support all of the listed values. The following standard
+  //   values are defined:
```

Comment:
Do you know if anyone has an existing list of ISA that we could point to? Maintaining our own list of values seems like it may be a maintenance burden in this case, if there's an easier way we can have an up-to-date list.

Reply:
Is it simply analogous to `uname -i` + extensions?

Reply:
> Do you know if anyone has an existing list of ISA that we could point to?

Unfortunately, I'm not aware of an existing list that would be suitable. The list in my patch aims to match official documentation where reasonable, but in a vendor-neutral and consistent form.

> Is it simply analogous to `uname -i` + extensions?

Do you mean `uname -m`? `uname -i` just prints 'unknown' here on Linux.

Unfortunately, `uname -m` is OS-specific. E.g., various BSDs use 'amd64' while many other operating systems use 'x86_64'. And Linux doesn't use anything generic for 32-bit x86, it reports 'i386', 'i486', 'i586' or 'i686', while SunOS/Solaris reports 'i86pc'. There are also inconsistencies for ARM.

Reply:
I've split the list into instruction set architectures and instruction set extensions/versions. Is the suggested list acceptable as a starting point or does anyone have an alternative suggestion?